src/roundups/325.md

```diff
@@ -60,7 +60,7 @@ language”_.
 located in the Netherlands _“mainly related to TV programs that are broadcast on 
 the channels of RTL Netherlands”_.
 
-As usual there has been a bunch of HaxeLib releases.
+As usual there has been a bunch of Haxelib releases.
```

Comment:
Good catch, this appears to be the official casing. I'll have to update the title refs as well and try to remember for future roundups.